themebuilder-scss/package.json

```diff
@@ -29,13 +31,20 @@
     "@types/clean-css": "^4.2.1",
     "@types/jest": "^25.2.1",
     "@types/node-sass": "^4.11.0",
-    "eslint": "^6.8.0",
+    "@typescript-eslint/eslint-plugin": "^2.33.0",
+    "eslint": "^7.0.0",
+    "eslint-config-airbnb-typescript": "^7.2.1",
+    "eslint-config-devextreme": "^0.1.0",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-jest": "^23.11.0",
+    "eslint-plugin-jest-formatting": "^1.2.0",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-react": "^7.20.0",
```

Comment:
@churkin  can we add these packages to `dependencies` of the `eslint-config-devextreme` package?